frontend/src/domain/cart/pages/Cart.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from '@emotion/styled';
+
+import { useCartContext } from '@/context/cart';
+
+export default function Cart() {
+  const { items } = useCartContext();
+
+  return (
+    <CartContainer>
+      <H2>장바구니 목록</H2>
+      <ul>
+        {items.map((item) => {
+          const { id, name, price, imageUrl } = item;
+
+          return (
+            <ProductDetailLi key={id}>
+              <LeftWing>
+                <img src={imageUrl} alt="item" />
+              </LeftWing>
+              <RightWing>
+                <div>{name}</div>
+                <div>{price}</div>
+              </RightWing>
+            </ProductDetailLi>
+          );
+        })}
```

Comment:
의도적으로 코드 라인 수를 줄인다면 가독성 향상에 좋습니다!

```suggestion
        {items.map(({ id, name, price, imageUrl }) => (
            <ProductDetailLi key={id}>
              <LeftWing>
                <img src={imageUrl} alt="item" />
              </LeftWing>
              <RightWing>
                <div>{name}</div>
                <div>{price}</div>
              </RightWing>
            </ProductDetailLi>
        )}
```